src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1058,26 +1058,48 @@ def get_active_child_names(self):
         """Names of children that are currently active."""
         ret = []
         for child in self.child_names:
-            if getattr(self, child).is_active():
+            if getattr(self, child).is_active() and not self._is_deprecated(child):
```

Comment:
@prmukherj This is seems blatantly illogical. We're saying these are the active children. The existing code asks each child if it's active. That's already logical and doesn't need to be changed; i.e. why add an additional predicate at this level? By doing it this way, we also get inconsistent behaviour between asking "which are the active children?" and "is this child active?".

Reply:
@prmukherj 

We have to give an accurate, consistent picture to the Python user, and that includes `obj.is_active()` and `obj.get_active_child_names()`. 

Is there a use case where we need a different `is_active()` behaviour?

Reply:
I don't think so. Yes, I agree, will update accordingly.